samigo/samigo-app/src/webapp/jsf/author/item/calculatedQuestion.jsp

```diff
@@ -46,9 +46,29 @@ confirmation dialog
 	<!-- AUTHORING -->
 	<script src="/samigo-app/js/authoring.js"></script>
 	<script>
-	$(document).ready(function() {
-		initCalcQuestion();
-	});
+		const addGlobalVariable = () => {
+			const globalvariable = prompt("<h:outputText value="#{authorMessages.calc_question_enter_global_variable}" escape="false"/>");
```

Comment:
THIS IS SUPER NICE and well supported.

![image](https://user-images.githubusercontent.com/8653754/223766045-16c820f1-b0ea-4b72-b05f-0ee7b0f7e956.png)

Reply:
Sure! :)